draft-irtf-cfrg-aegis-aead.md

```diff
@@ -1773,20 +1775,20 @@ It finalizes all the instances, absorbs the resulting tags into the first state,
 
 ~~~
 t = {}
-u = LE64(data_len_bits) || LE64(tag_length_bits)
+u = LE64(data_len_bits) || LE64(tag_len_bits)
 for i in 0..D:
     t = t || (V[3,i] ^ u)
 
 Repeat(7, Update(t))
 
 tags = {}
-if tag_length_bits == 128:
+if tag_len_bits == 128:
     for i in 1..D:   # tag from state 0 is skipped
         ti = V[0,i] ^ V[1,i] ^ V[2,i] ^ V[3,i] ^ V[4,i] ^ V[5,i]
         tags = tags || ti
 
-else:                # 256 bits
-    for i in 1..D:   # tag from state 0 is skipped
+else:              # 256 bits
```

Comment:
Think these comments need to be shifted right to match the top one.

Reply:
The diff looks misleading; everything seems to be aligned:

```md
if tag_len_bits == 128:
    for i in 1..D: # tag from state 0 is skipped
        ti = V[0,i] ^ V[1,i] ^ V[2,i] ^ V[3,i] ^ V[4,i] ^ V[5,i]
        tags = tags || ti

else:              # 256 bits
    for i in 1..D: # tag from state 0 is skipped
```